tox.ini

```diff
@@ -1,95 +1,21 @@
+[tox]
```

Comment:
Updated the tox file to fix the issue where travis was not able to install the correct version of django and to simplify things.